Sources/Commands/SwiftPackageTool.swift

```diff
@@ -305,6 +305,14 @@ extension SwiftPackageTool {
         @OptionGroup(_hiddenFromHelp: true)
         var swiftOptions: SwiftToolOptions
 
+        @Option(help: """
+        The path to a text file containing breaking changes which should be ignored by the API comparison. \
+        Each ignored breaking change in the file should appear on its own line and contain the exact message \
+        to be ignored (e.g. 'API breakage: func foo() has been removed'). If this option is not present, \
+        a file named 'api-breakage-allowlist.txt' at the package root will be used if it exists.
```

Comment:
The default behavior doesn't seem intuitive for this input option and it would be cleaner to just require a specific path to be passed in by a user.

Reply:
sounds good to me!